### PR TITLE
webkit2gtk, seatd, libassuan, libgcrypt, and ruby bumped

### PR DESCRIPTION
**Pull request by muevoid**
# Existing Package

- [x] I am the maintainer of this package.

### CODE REVIEW

File: community/ruby/sources [hunk 1]
```diff
@@ -1 +1 @@
-https://cache.ruby-lang.org/pub/ruby/2.7/ruby-2.7.2.tar.gz
+http://cache.ruby-lang.org/pub/ruby/3.0/ruby-3.0.0-preview1.tar.xz
```
Comment (dylanaraps) [inline community/ruby/sources:1]: Why drop https? Also, preview release(?)

**Comment by muevoid** [comment]
In relationship to the https that is my bad and I can fix it. Otherwise yes it is preview however it was in the link you sent me so I can remove that commit altogether if youd like.

**Comment by muevoid** [comment]
Sorry if there was a misunderstanding which one would you prefer? Dropping this update for now or just fix https?

**Comment by dylanaraps** [comment]
Drop the ruby update.

**Comment by muevoid** [comment]
fixed

**Pull request merged**